reference/customers.v3.yml

```
@@ -3359,6 +3359,94 @@ components:
         - country_code
       description: The `address` object for the `customer` object's `addresses` array.
       x-internal: false
+    customerAddresses_CustomerPost:
+      title: customerAddresses_CustomerPost
+      example:
+        address1: Addr 1
+        address2: ''
+        address_type: residential
+        city: San Francisco
+        company: History
+        country_code: US
+        first_name: Ronald
+        last_name: Swimmer
+        phone: '707070707'
+        postal_code: '33333'
+        state_or_province: California
+        form_fields: 
+          - name: "test"
+            value: "test"
+      type: object
+      properties:
+        first_name:
+          description: The first name of the customer address.
```

These are not quite right... the name doesn't belong to the address. but it also might not directly belong to the customer? could be a gift recipient, for example. Reflect & consider revising.

---

@bc-akshatkaushik can the description be reworded to better reflect the object type?

---

@slsriehl So the address first name is separate to the customer first name, for example I as a customer can have an address with my friend's details if I want to have something delivered to him.
Do you still think a revision is required?

---

@bc-akshatkaushik this is helpful, thanks. It's not so much that a revision is required; it's that one of our value adds as a team is to optimize language to leave the reader with no feeling of loose ends.

@bc-tgomez and also Akshat, how do you feel about something like the following?

```suggestion
          description: The first name associated with the customer address.
```

---

I agree @slsriehl that is definitely very important and something we tend to miss out on as developers.
Your suggested change is more in line with the internal implementation and intention of use for the field, looks good to me.
Thank You for pointing it out, looking forward to more of the same as we are in the process of planning to audit the customers devdocs space for inconsistencies with what is implemented.